src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVCleanPt1a.java

```diff
@@ -0,0 +1,439 @@
+ * <ul>
+ *     <li>
+ *         VCF containing structural variant (SV) records from the GATK-SV pipeline.
+ *     </li>
+ *     <li>
+ *         TODO
+ *     </li>
+ * </ul>
+ *
+ * <h3>Output</h3>
+ * <ul>
+ *     <li>
+ *         Annotated VCF.
+ *     </li>
+ * </ul>
+ *
+ * <h3>Usage Example</h3>
+ * <pre>
+ *     gatk SVCleanPt1a \
+ *       -V structural.vcf.gz \
+ *       -O cleansed.vcf.gz
+ *       --ped-file pedigree.ped
+ *       --chrX chrX
+ *       --chrY chrY
+ *       --fail-list background_fail.txt
+ *       --pass-list bothsides_pass.txt
+ *       --sample-list sample_list.txt
+ *       --revised-list revised_list.txt
+ * </pre>
+ *
+ * <h3>Cleaning Steps</h3>
+ * <ol>
+ *     <li>
+ *         Adds new FILTER and INFO tags to header.
+ *     </li>
+ *     <li>
+ *         TODO
+ *     </li>
+ * </ol>
+ */
+@CommandLineProgramProperties(
+        summary = "Clean and format structural variant VCFs per Step 1a",
+        oneLineSummary = "Clean and format structural variant VCFs per Step 1a",
+        programGroup = StructuralVariantDiscoveryProgramGroup.class
+)
+@BetaFeature
+@DocumentedFeature
+public final class SVCleanPt1a extends VariantWalker {
+    public static final String PED_FILE_LONG_NAME = "ped-file";
+    public static final String CHRX_LONG_NAME = "chrX";
+    public static final String CHRY_LONG_NAME = "chrY";
+    public static final String FAIL_LIST_LONG_NAME = "fail-list";
+    public static final String PASS_LIST_LONG_NAME = "pass-list";
+    public static final String OUTPUT_SAMPLES_LIST_LONG_NAME = "sample-list";
+    public static final String OUTPUT_REVISED_EVENTS_LIST_LONG_NAME = "revised-list";
+
+    @Argument(
+            fullName = PED_FILE_LONG_NAME,
+            doc = "Sample PED file"
+    )
+    private GATKPath pedFile;
+
+    @Argument(
+            fullName = CHRX_LONG_NAME,
+            doc = "chrX column name"
+    )
+    private String chrX;
+
+    @Argument(
+            fullName = CHRY_LONG_NAME,
+            doc = "chrY column name"
+    )
+    private String chrY;
+
+    @Argument(
+            fullName = FAIL_LIST_LONG_NAME,
+            doc = "List of complex variants failing the background test"
+    )
+    private GATKPath failList;
+
+    @Argument(
+            fullName = PASS_LIST_LONG_NAME,
+            doc = "List of complex variants passing both sides"
+    )
+    private GATKPath passList;
+
+    @Argument(
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc = "Output VCF name"
+    )
+    private GATKPath outputVcf;
+
+    @Argument(
+            fullName = OUTPUT_SAMPLES_LIST_LONG_NAME,
+            doc="Output list of samples"
+    )
+    private GATKPath outputSamplesList;
+
+    @Argument(
+            fullName = OUTPUT_REVISED_EVENTS_LIST_LONG_NAME,
+            doc="Output list of revised genotyped events"
+    )
+    private GATKPath outputRevisedEventsList;
```

Comment:
Will it be possible to build this info directly into the VCF rather than have this extra file?

Reply:
Updated accordingly - added this as a new flag in the INFO field.